app/src/components/common/layout/header/index.tsx

```diff
@@ -166,6 +166,24 @@ const HeaderDropdown = styled(Dropdown)`
   height: 40px;
 `
 
+const MarketAndGovernanceNav = styled.div<{ disabled?: boolean }>`
+  width: 124.65px;
```

Comment:
I think you don't need to define width with static width values dunno what @pimato thinks but according to designs width of markets is smaller..

Reply:
if i'm not mistaken, the width is only different because the length of text is different, and that's auto generated by figma. the width of the pill button design is defined as `width: 124px;`

Reply:
nevermind you're right, good call